components/feature/search/src/main/assets/extensions/search/manifest.template.json

```diff
@@ -0,0 +1,220 @@
+        "https://www.google.com.pa/search*",
+        "https://www.google.com.pe/search*",
+        "https://www.google.com.pg/search*",
+        "https://www.google.com.ph/search*",
+        "https://www.google.com.pk/search*",
+        "https://www.google.pl/search*",
+        "https://www.google.pn/search*",
+        "https://www.google.com.pr/search*",
+        "https://www.google.ps/search*",
+        "https://www.google.pt/search*",
+        "https://www.google.com.py/search*",
+        "https://www.google.com.qa/search*",
+        "https://www.google.ro/search*",
+        "https://www.google.ru/search*",
+        "https://www.google.rw/search*",
+        "https://www.google.com.sa/search*",
+        "https://www.google.com.sb/search*",
+        "https://www.google.sc/search*",
+        "https://www.google.se/search*",
+        "https://www.google.com.sg/search*",
+        "https://www.google.sh/search*",
+        "https://www.google.si/search*",
+        "https://www.google.sk/search*",
+        "https://www.google.com.sl/search*",
+        "https://www.google.sn/search*",
+        "https://www.google.so/search*",
+        "https://www.google.sm/search*",
+        "https://www.google.sr/search*",
+        "https://www.google.st/search*",
+        "https://www.google.com.sv/search*",
+        "https://www.google.td/search*",
+        "https://www.google.tg/search*",
+        "https://www.google.co.th/search*",
+        "https://www.google.com.tj/search*",
+        "https://www.google.tl/search*",
+        "https://www.google.tm/search*",
+        "https://www.google.tn/search*",
+        "https://www.google.to/search*",
+        "https://www.google.com.tr/search*",
+        "https://www.google.tt/search*",
+        "https://www.google.com.tw/search*",
+        "https://www.google.co.tz/search*",
+        "https://www.google.com.ua/search*",
+        "https://www.google.co.ug/search*",
+        "https://www.google.co.uk/search*",
+        "https://www.google.com.uy/search*",
+        "https://www.google.co.uz/search*",
+        "https://www.google.com.vc/search*",
+        "https://www.google.co.ve/search*",
+        "https://www.google.vg/search*",
+        "https://www.google.co.vi/search*",
+        "https://www.google.com.vn/search*",
+        "https://www.google.vu/search*",
+        "https://www.google.ws/search*",
+        "https://www.google.rs/search*",
+        "https://www.google.co.za/search*",
+        "https://www.google.co.zm/search*",
+        "https://www.google.co.zw/search*",
+        "https://www.google.cat/search*",
+        "https://www.baidu.com/*",
+        "https://m.baidu.com/*",
+        "https://*search.yahoo.com/search*",
+        "https://www.bing.com/search*",
+        "https://duckduckgo.com/*"
+      ],
+      "js": ["searchTelemetry.js"],
+      "run_at": "document_end"
+    }
+  ],
+  "permissions": [
+    "geckoViewAddons",
+    "nativeMessaging",
+    "nativeMessagingFromContent",
+    "*://*/*"
```

Comment:
Is it possible to limit to the domains above instead of "*://*/*" ?

Reply:
The documentation - https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions is not really great and I'm not sure of the support though I can try testing with the same array from `include_globs`.

But if the extension can have these permissions on all sites but it is active only on specific ones, do we really need to filter them again?

Reply:
Yes, we can add host permissions using the same match patterns, but since you tightened the content_script matchers, we can also do that separately and not pile everything on here :).

Reply:
I have added all addresses used in the above `include_globs` array in the `permissions` block also.
In my tests everything seems to work as expected (saw telemetry for each search engine (only tried one google variant)).

Reply:
👍